README.md

```diff
@@ -0,0 +1,32 @@
+# Facebook for WooCommerce
+
+This is the development repository for the Facebook for WooCommerce plugin.
+
+- [WooCommerce.com product page](https://woocommerce.com/products/facebook)
+- [WordPress.org plugin page](https://wordpress.org/plugins/facebook-for-woocommerce/)
+- [User documentation](https://docs.woocommerce.com/document/facebook-for-woocommerce)
+
+## Support
+The best place to get support is the [WordPress.org Facebook for WooCommerce forum](https://wordpress.org/support/plugin/facebook-for-woocommerce/).
+
+If you have a WooCommerce.com account, you can [open a ticket on WooCommerce.com](https://woocommerce.com/my-account/create-a-ticket/).
+
+## Development 
+### Developing
+- Clone this repository into the `wp-content/plugins/` folder your WooCommerce development environment.
+- Install dependencies: 
+	- `npm install`
+	- `composer install`
+- Build assets:
+	- `npm start` to build a development version
```

Comment:
@danielbitzer I've added this since #1826 is going to merge soon 🎉 

Feel free to update this PR with better/more/clearer instructions (depending on how scripts are set up on that branch).